scripts/exregional_make_ics.sh

```diff
@@ -548,6 +549,26 @@ case "${EXTRN_MDL_NAME_ICS}" in
   convert_nst=False
   ;;
 
+"RRFS")
+  external_model="RRFS"
+
+  fn_grib2="${EXTRN_MDL_FNS[0]}"
+  input_type="grib2"
+#
+# Path to the HRRRX geogrid file.
+#
+  geogrid_file_input_grid="${FIXgsm}/geo_em.d01.nc_HRRRX"
```

Comment:
Is this file actually used? Geogrid is a WRF- and MPAS-specific utility as far as I'm aware.

Reply:
It could be removed now, as RRFS is a fully supported ICS/LBCS format. It remained from early tests with RRFS data, when it had to be labeled as "HRRR" to work with the pre-RRFS UFS_UTILS.